st2auth_pam_backend/pam_backend.py

```diff
@@ -38,7 +44,9 @@ class PAMAuthenticationBackend(object):
     """
 
     def __init__(self):
-        pass
+        uid = os.geteuid()
```

Comment:
I will also add an argument to the constructor so users can disable this behavior in case they have a good reason for that (e.g. user under which st2auth is running is already in wheel group and doesn't need to be root or similar).